src/SFA.DAS.ContentApi.Database/Tables/Client.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Client]
+(
+	[Id] INT NOT NULL PRIMARY KEY, 
```

Comment:
It's very common to use UNIQUEIDENTIFIER for primary keys as having one in a table is a requirement for merge replication - is there a good reason not too?

Reply:
Why would this table use a UNIQUEIDENTIFIER and not the others?

Reply:
The comment applies to the whole database

Reply:
Using int here is consistent with other MA application; EPAO is the different version and since there is no official replication strategy will assume using int will be ok and is consistent.